peer/brontide.go

```diff
@@ -2611,20 +2613,19 @@ func (p *Brontide) writeMessage(msg lnwire.Message) error {
 //
 // NOTE: This method MUST be run as a goroutine.
 func (p *Brontide) writeHandler() {
-	// We'll stop the timer after a new messages is sent, and also reset it
-	// after we process the next message.
-	idleTimer := time.AfterFunc(idleTimeout, func() {
-		err := fmt.Errorf("peer %s no write for %s -- disconnecting",
-			p, idleTimeout)
-		p.Disconnect(err)
-	})
-
 	var exitErr error
 
+	// Create a timer to time out the write. We create it here to avoid
+	// allocations in the loop below.
+	flushTimer := time.NewTimer(writeFlushTimeout)
+
 out:
 	for {
 		select {
 		case outMsg := <-p.sendQueue:
+			// Reset the timeout.
+			flushTimer.Reset(writeFlushTimeout)
```

Comment:
Given we are using go1.23, there's no need to call `Stop` anymore, [reference](https://antonz.org/go-1-23/#timer-changes).